defaults/main.yml

```diff
@@ -23,6 +23,10 @@ galaxy_tools_base_dir: /tmp
 # Blank variable to make sure it's defined
 galaxy_tools_api_key: ''
 
+# List of workflow files to be installed
+
+galaxy_tools_workflows: []
```

Comment:
I think it'd be good to also include a sample of the values that would be used to populate this list.